server.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+// npm modules
+const cors = require('cors');
+const morgan = require('morgan');
+const express = require('express');
+const debug = require('debug')('apple:server');
+const mongoose = require('mongoose');
+const Promise = require('bluebird');
+
+// app modules
+const listRouter = require('./route/list-route.js');
+
+// module constants
+const PORT = process.env.PORT || 3000;
+const MONGODB_URI = process.env.MONGODB_URI || 'mongodb://localhost/appledev';
+
+// connect to database
+mongoose.Promise = Promise;
+mongoose.connect(MONGODB_URI);
+
+const app = express();
+app.use(cors());
+app.use(morgan('dev'));
+
+
+app.use(listRouter);
+
+const server = module.exports = app.listen(PORT, function(){
+  debug(`server up ${PORT}`);
+});
+
+server.isRunning = true;
```

Comment:
@loganlsr not sure if you have done this yet but be sure to create an environment variable for your mongo URI, that way this doesn't always default to the hard-coded string that will be used in the event that an environment variable has not been found